fixtures/glp/impression-x4-bar-10.json

```diff
@@ -0,0 +1,994 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Impression X4 Bar 10",
+  "categories": ["Moving Head", "Color Changer", "Effect", "Strobe"],
+  "meta": {
+    "authors": ["Rummels Bucht Berlin"],
+    "createDate": "2018-05-19",
+    "lastModifyDate": "2018-05-19"
+  },
+  "comment": "impression X4 BAR 10 DMX chart (v1.6) from software version: v0.60",
+  "manualURL": "https://www.glp.de/en/products?format=raw&task=download&fid=112",
+  "physical": {
+    "dimensions": [500, 240, 100],
+    "weight": 8,
+    "power": 200,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "Osram Ostar RGBW multi-chip LED",
+      "colorTemperature": 10000
+    },
+    "focus": {
+      "tiltMax": 210,
+      "type": "Head"
+    }
+  },
+  "matrix": {
+    "pixelCount": [10, 1, 1],
+    "pixelGroups": {
+      "1 & 2": ["1", "2"],
+      "3 & 4": ["3", "4"],
+      "5 & 6": ["5", "6"],
+      "7 & 8": ["7", "8"],
+      "9 & 10": ["9", "10"]
+    }
+  },
+  "availableChannels": {
+    "Tilt": {
+      "type": "Tilt",
+      "fineChannelAliases": ["Tilt fine"],
+      "defaultValue": 32512,
```

Comment:
32512 results in 127/0 for coarse/fine channels. Shouldn't it be 32767 for 127/255 (which is closest to 50%)?

Reply:
I'll use 32768, which is 128/0, so it works fine in the non-fine channel as well.